jwt/utils.py

```diff
@@ -51,3 +61,51 @@ def uint_b64decode(uint_b64: str) -> int:
         value <<= 8
         value += int(b)
     return value
+
+
+def get_time(value: Union[str, int, datetime]) -> Union[int, datetime, None]:
```

Comment:
This use of Union in the return forces the caller to do type switch or the caller can call only methods implemented on all of `int`, `datetime` and `None` for type safety. If we implement helpers for converting `datetime`, I think it is better to implement methods for each return types.

Reply:
Ok, i will do it